src/vs/workbench/contrib/terminal/browser/terminalTabbedView.ts

```diff
@@ -176,6 +194,80 @@ export class TerminalTabbedView extends Disposable {
 		}
 	}
 
+	private _createChatTerminalsEntry(container: HTMLElement): void {
+		this._chatTerminalsEntry = dom.append(container, $('.terminal-tabs-chat-entry.monaco-list-row'));
+		this._chatTerminalsEntry.tabIndex = 0;
+		this._chatTerminalsEntry.setAttribute('role', 'button');
+
+		const entry = dom.append(this._chatTerminalsEntry, $('.terminal-tabs-entry'));
+		const icon = dom.append(entry, $('.terminal-tabs-chat-entry-icon'));
+		icon.classList.add(...ThemeIcon.asClassNameArray(Codicon.commentDiscussion));
+		this._chatTerminalsLabel = dom.append(entry, $('.terminal-tabs-chat-entry-label'));
+		this._chatTerminalsCount = dom.append(entry, $('.terminal-tabs-chat-entry-count'));
+
+		const runChatTerminalsCommand = () => {
+			void this._commandService.executeCommand('workbench.action.terminal.chat.viewChatTerminals');
+		};
+
+		this._register(dom.addDisposableListener(this._chatTerminalsEntry, dom.EventType.CLICK, e => {
+			e.preventDefault();
+			runChatTerminalsCommand();
+		}));
+		this._register(dom.addDisposableListener(this._chatTerminalsEntry, dom.EventType.KEY_DOWN, e => {
+			if (e.key === 'Enter' || e.key === ' ') {
+				e.preventDefault();
+				runChatTerminalsCommand();
+			}
+		}));
+	}
+
+	private _updateChatTerminalsEntry(): void {
```

Comment:
Seems like there's enough here to make a widget class and manage everything inside it, otherwise it increases the complexity of `TerminalTabbedView` quite a bit.